circle.yml

```diff
@@ -7,6 +7,8 @@ machine:
 dependencies:
   bundler:
     without: [oracle]
+  pre:
+    - cp config/secrets.yml.template config/secrets.yml
```

Comment:
We tend to use secrets/settings files over environment variables (which is what circle can set). In some setups, you commit the secrets file, but have it load environment variables. We've never gone down this route.

One thing this does is validates the template has all the things we need. We don't want nil errors, especially if you're going multi levels down in the secrets tree.